oslabs-beta/Kubiquity
# Connect FE to BE (naive solution)

WHAT DOES THIS PR DO? 
1. Add a testing route for errors (so that we do not have to have a K8s cluster running to get errors from the backend)
2. Add MIT license. 

HOW TO TEST THIS PR:
1. Run the following command: `gh pr checkout 14 && npm install && npm run dev`
2. ONCE WEBPACK HAS CREATED OUR NEW BUILD, AND ONCE THE SERVER IS RUNNING, AND ONCE DB IS CONNECTED, verify that the Errors table is populated with a lot of errors. 

ADDITIONAL NOTES: 
1. JEFF and or DAVID: can you confirm that all these errors are unique?
2. as noted, this is a very naive solution—we're literally querying localhost in fetch. However, because this likely will change once we've converted our app to fully use Electron, it seems like a fine middle ground between working and not working. 
3. Since we already have thousands of errors, we'll have to add in some sort of way to deal with this—on the FE, this might be as simple as paginating the table, but perhaps worthy of a full team meeting.
